api/SpeechRecognitionErrorEvent.json

```diff
@@ -5,28 +5,16 @@
         "mdn_url": "https://developer.mozilla.org/docs/Web/API/SpeechRecognitionErrorEvent",
         "spec_url": "https://wicg.github.io/speech-api/#speechrecognitionerrorevent",
         "support": {
-          "chrome": [
-            {
-              "version_added": "77",
-              "notes": "You'll need to serve your code through a web server for recognition to work."
-            },
-            {
-              "alternative_name": "webkitSpeechRecognitionError",
-              "version_added": "33",
-              "version_removed": "77"
-            }
-          ],
-          "chrome_android": [
-            {
-              "version_added": "77",
-              "notes": "You'll need to serve your code through a web server for recognition to work."
-            },
-            {
-              "alternative_name": "webkitSpeechRecognitionError",
-              "version_added": "33",
-              "version_removed": "77"
-            }
-          ],
+          "chrome": {
+            "alternative_name": "webkitSpeechRecognitionError",
+            "version_added": "33",
+            "notes": "Since Chrome 77, you'll need to serve your code through a web server for recognition to work."
```

Comment:
This note is all over the speech recognition APIs but I'd like to trim to to just the main entry point for the API.

There actually isn't anything worth noting about Chrome 77. This data was added in https://github.com/mdn/browser-compat-data/pull/5584 with reference to https://github.com/chromium/chromium/commit/5dbd1fc, but all that did was to change the name one sees when doing things like stringifying an instance. `webkitSpeechRecognitionError` is still the only way to get the interface object without an instance.

So let's get rid of the Chrome 77 stuff and the notes.

@sideshowbarker FYI